webpack.dev.config.js

```diff
@@ -1,11 +1,11 @@
-const path = require('path');
 const { createConfig } = require('@edx/frontend-build');
+const path = require('path');
```

Comment:
Similar note here to the jest config, I would defer to what is already in `master` since I fixed some eslint issues and standardized the import orders between these 2 files.